schema/xsd/portal.xsd

```diff
@@ -59,13 +59,21 @@
         </xs:choice>
     </xs:complexType>
 
+    <xs:complexType name="link-notification">
+        <xs:choice minOccurs="1" maxOccurs="1">
+            <xs:element name="email" type="email" minOccurs="1" maxOccurs="1" />
```

Comment:
`minOccurs="1"` og `maxOccurs="1"` er forøvrig standardverdier (er jeg 90+% sikker på 💃 ), så du kan vurdere om du vil fjerne det, eller ha det med for å være eksplisitt.